Model/CustomConfigProvider.php

```diff
@@ -25,13 +25,15 @@ public function getConfig()
     {
         $config = $this->getScopeConfig()->getValue('carriers/parcelpro', \Magento\Store\Model\ScopeInterface::SCOPE_STORE);
 
-        $gebruikerid = $config["gebruiker_id"];
+        if (isset($config["gebruiker_id"])) {
+            $gebruikerid = $config["gebruiker_id"];
+            $config = [
+                'config' => [
+                    'gebruikerID' => $gebruikerid
+                ]
+            ];
+        }
```

Comment:
In andere stukken code is deze variable ook nog steeds niet in gebruik. Zou het niet eerder een idee zijn om de plugin niks te laten doen als gebruikers_id niet is ingevult.

Reply:
Yea, opzich mee eens, maar dit is concreet een plek die (blijkbaar) errors veroorzaakt voor een van de klanten, waar zij vervolgens op hun server een patch op zetten. Dus dit leek me een mooi stukje low-hanging fruit.

Reply:
Laten we dat dan een andere keer doen en het voor nu bij dit laten zodat de klant ermee aan de slag kan.